ods_ci/tests/Tests/100__deploy/120__upgrades/120__pre_upgrades.robot

```diff
@@ -63,7 +76,36 @@ Verify Custom Image Can Be Added
     [Tags]  Upgrade
     Oc Apply    kind=ImageStream   src=ods_ci/tests/Tests/100__deploy/120__upgrades/custome_image.yaml
 
+Verify User Can Disable The Runtime
+    [Documentation]  Disable the Serving runtime using Cli
+    [Tags]  Upgrade
+    Disable Model Serving Runtime Using CLI   namespace=redhat-ods-applications
 
+Verify Model Can Be Deployed Via UI For Upgrade
+    [Tags]  Upgrade
+    [Setup]    Begin Web Test
+    Fetch CA Certificate If RHODS Is Self-Managed
+    Open Model Serving Home Page
+    Try Opening Create Server
+    Wait for RHODS Dashboard to Load    wait_for_cards=${FALSE}    expected_page=Data science projects
+    Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
```

Comment:
Could we reuse the same data science project for all the tests like pipelines, etc., ?

Reply:
I wouldn't do that it will create issue it should be independent.